capsicum-test.cc

```diff
@@ -50,7 +50,11 @@ char ProcessState(int pid) {
 #endif
 #ifdef __FreeBSD__
   char buffer[1024];
-  snprintf(buffer, sizeof(buffer), "ps -p %d -o state | grep -v STAT", pid);
+  /* 
+   * XXX: -a is used with ps(1) as a temporary workaround for retrieving the
+   * state of zombie processes
```

Comment:
Could you add a summary of what/why/how this is working around?  Otherwise this is going to make no sense to me when I come across it again in a year or two's time :-)